### DISCUSSION TITLE
How to remove the progress bar when task finished

### DISCUSSION THREAD

**Discussion by Wsine** [Textualize/rich#1102] [Q&A] [answered]
The same feature as `tqdm(..., leave=True)`

**Comment by willmcgugan** [comment, marked as answer]
https://rich.readthedocs.io/en/latest/progress.html#transient-progress

**Reply by Wsine** [reply]
how if I have multiple progress bars via `add_task` from `Progress(transient=True)`? Currently, only if all the tasks finished, these progress bars will disappear together. 

Actually, I dont want them to occupy my screen. just to disappear once they finish.

Is that possible now?

**Reply by willmcgugan** [reply]
You can remote a task https://rich.readthedocs.io/en/latest/reference/progress.html#rich.progress.Progress.remove_task

Or update it with visible=False to make it temporarily invisible.

Please do have a look through the docs.
